### DISCUSSION TITLE
How to use “auth” without a hash password?

### DISCUSSION THREAD

**Discussion by septasite** [adonisjs/core#2609] [Help] [answered]
i try login user without hash password but password mismatches
I think maybe auth.attempt only work with hash password.

could there be another way to use auth without hash psswrd?

i use adonis 4.1

**Comment by RomainLanz** [comment, marked as answer]
Hey @septasite! 👋 

They are plenty of other ways, and they are all explained in our documentation.
Also, not hashing a password is a VERY BAD idea.

▶ https://legacy.adonisjs.com/docs/4.1/authentication

**Comment by septasite** [comment]
solved
